e2e-tests/oauth-flow/rfc021/node-A/presentationexchangemapping.json

```diff
@@ -91,6 +91,17 @@
                 "filter": {
                   "type": "string"
                 }
+              },
+              {
+                "id": "employee_role_level",
```

Comment:
was the e2e test supposed to test the capture group?

Reply:
probably not the place to test it. Unit test is enough imo. Reverted it.